cmake/SetupMFMG.cmake

```diff
@@ -12,12 +12,12 @@ set(CMAKE_CXX_FLAGS_DEBUG "-g")
 
 # Disable boost asserts in Release mode and set MFMG_DEBUG in Debug mode
 MESSAGE("-- Build type: ${CMAKE_BUILD_TYPE}")
-IF(CMAKE_BUILD_TYPE MATCHES "Release")
+IF(CMAKE_BUILD_TYPE MATCHES "Release" OR CMAKE_BUILD_TYPE MATCHES "RelWithDebInfo")
```

Comment:
I think that for `RelWithDebInfo` we should set both `-O3` and `-g` flags. Otherwise it is not in a release build and there is no debug info.

Reply:
CMake does it for you.

Reply:
OK great.